src/dvclive/live.py

```diff
@@ -370,14 +371,25 @@ def log_metric(
         logger.debug(f"Logged {name}: {val}")
 
     def log_image(self, name: str, val):
-        if not Image.could_log(val):
-            raise InvalidDataTypeError(name, type(val))
```

Comment:
WDYT about this instead?
1. Leaving `could_log` as is (keep it here and don't change the method)
2. Where you moved `could_log` below, check the name suffix(`if not Path(name).suffix in Image.suffixes`)

It's less changes to the existing codebase, and I think it would be better to have separate errors for invalid image input values and invalid output names.